nixos/modules/security/sudo.nix

```diff
@@ -42,6 +42,37 @@ in
         '';
     };
 
+    security.sudo.enableSecurePath = mkOption {
+      type = types.bool;
+      default = true;
```

Comment:
Options considered harmful. If we do this, it should not be an option IMHO. Random variability between NixOS systems should be avoided.

Reply:
+1, since it can be overridden with extraRules anyway.